app/views/RoomsListView/index.js

```diff
@@ -526,6 +532,13 @@ export default class RoomsListView extends LoggedView {
 					keyboardShouldPersistTaps='always'
 					initialNumToRender={9}
 					windowSize={9}
+					refreshControl={(
+						<RefreshControl
+							// eslint-disable-next-line react/destructuring-assignment
```

Comment:
there is no need in disabling eslint here, just use state destructuring on line 512: `const { loading, refreshing } = this.state;`

Reply:
Thanks :)